config/webpacker.yml

```diff
@@ -10,7 +10,7 @@ default: &default
 
   # Additional paths webpack should lookup modules
   # ['app/assets', 'engine/foo/app/assets']
-  additional_paths: ['node_modules/govuk-frontend/govuk', 'node_modules/govuk_frontend_toolkit']
+  additional_paths: ['node_modules/govuk-frontend/govuk', 'node_modules/govuk_frontend_toolkit', 'spec/javascripts']
```

Comment:
Does `spec/javascripts` need to be added or is this redundant, especially as you are already loading all js specs via entry file `app/webpack/packs/specs.js`

Reply:
refer to the documentation step 5: [Jasmine](https://jasmine.github.io/setup/browser.html#use-with-rails)